packages/react-centra-checkout/package.json

```diff
@@ -34,10 +34,12 @@
     "test-types": "tsc --noEmit"
   },
   "dependencies": {
+    "@noaignite/utils": "workspace:*",
```

Comment:
todo: Now that this is actually a dependency and not a devDep I wonder if this should not actually have the current version of the `@noaignite/types` package. The changeset-bot does actually update these via the `changeset version` script when it creates the `Version Packages` PR.